developers/verifiers/multicredentials.md

```diff
@@ -20,34 +20,72 @@ Here's how to get a demo Veres Wallet if you need one:
+Now, lets issue the credential. Click "Issue Verifiable Credential" to proceed to the (optional) DID Authentication screen.
 
-## Multiple VC Verification
+Once you've completed DID Authentication (or not), you'll be provided with the Verifiable Credential JSON.
 
-Now that you have several VCs in your wallet, let's get them all verified in one request.
+{% include example-issued-alumni-credential.html %}
 
-1. Visit the playground's Verifier section at <https://playground.chapi.io/verifier>.
+**NOTE:** The credential JSON document now has an absolute URL for it's `id` value and a new `proof` section. The `issuer.id` has also been updated to reflect the playground as the Issuer, and (of course) the `issuanceDate` has also been set.
 
-2. Click each of the VCs you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF2", and "Permanent Resident" for example).
+That credential may now be stored in a wallet. Click "Store in Wallet" to proceed.
 
-3. Click "Request Verifiable Presentation" to request that your wallet share these credentials.
+A modal will appear - similar to the one below - explaining that the playground wants to send the cerdential for you.
 
-4. In the demo Veres Wallet you'll see each of the credentials you selected (plus the ability to select which ones you want to verify). Click the "Share" button to send them to the playground for verification.
+![Playground offering to store a credential. The screen contains two buttons: one to cancel the request and one to go to the next screen.](playground-offering-to-store-credential.jpg)
 
-5. The process should end with a VerifiablePresentation JSON document and the words "Presentation Verified".
+Clicking "Next" should present you with a list of available wallets:
+
+![Choose a wallet modal presenting all preregisterd wallet systems which can be clicked on to proceed to storing the credentials there.](choose-a-wallet.jpg)
+
+However, if you instead receive a browser window popup warning you that you do not yet have one set up, you will need to return to the [Wallet Sign Up section](#wallet-sign-up) above.
+
+If you received a list of wallet options, pick the one you prefer by clicking on it's name to proceed.
+
+Each wallet will have it's own system for displaying the credential and offering to store it for you.
+
+Once you've clicked "Store" in your preferred wallet system, you will be brought back to the main playground screen.
+
+Follow the above process a few more times by clicking "Issuer Demo" to load your wallet with more example credentials.
+
+## Multiple Verifiable Credential Verification
+
+Now that you have several Verifiable Credentials (VCs) in your wallet, let's get them all verified in one request.
+
+First, visit the [playground's Verifier section](https://playground.chapi.io/verifier).
+
+Select the VCs you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF2", and "Permanent Resident").
```

Comment:
```suggestion
Select the VCs you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF3", and "Permanent Resident").
```

Reply:
I think that text is what went out in the email prior to the PF3 credential was ready. I'll update.